data/js/item/halloweenmasks.js

```diff
@@ -0,0 +1,30 @@
+function onCreateDFN( objMade, objType ) 
+{
+	var randomnamelist2009 = [
```

Comment:
Not a now problem, but ideally there'd be a way to look this up from a list in a DFN somewhere - particularly given namelists.dfn does exist